src/main/java/org/retest/rebazer/connector/GithubConnector.java

```diff
@@ -81,6 +88,14 @@ public boolean greenBuildExists( final PullRequest pullRequest ) {
 		return getGitHubChecks( pullRequest ).stream().allMatch( "success"::equals );
 	}
 
+	public String newestChecksTime( final PullRequest pullRequest ) {
+		return getGitHubChecksTimeStamps( pullRequest ).stream()//
+				.filter( time -> time != null && !time.isEmpty() )//
```

Comment:
```suggestion
				.filter( StringUtils::isNotEmpty )//
```

Reply:
Is there a reason why this has been resolved?

Reply:
In order to use StringUtils :: isNotEmpty, I have to add a dependency to the project and we have decided verbally that it would not be worth it